src/data/input_output/norm_mappers.py

```diff
@@ -0,0 +1,70 @@
+from utils import CoreMapper
+from utils.enum import Result, BoardColor
+
+
+class NormResult(CoreMapper[str, Result]):
+    @staticmethod
+    def _core_object_by_outer_value() -> dict[str, Result]:
+        return {
+            '0': Result.LOSS,
+            '0.5': Result.DRAW,
+            '1': Result.WIN,
+        }
+
+    @classmethod
+    def get_outer_value(cls, core_object: Result) -> str:
+        if norm_result := super().get_outer_value(core_object):
+            return norm_result
+        norm_result_by_result = {
+            Result.UNRATED_LOSS: '-',
+            Result.UNRATED_DRAW: '=',
+            Result.UNRATED_WIN: '+',
+            Result.FORFEIT_LOSS: '-',
+            Result.FORFEIT_WIN: '+',
+            Result.HALF_POINT_BYE: '=',
+            Result.FULL_POINT_BYE: '+',
+            Result.PAIRING_ALLOCATED_BYE: '+',
+            Result.ZERO_POINT_BYE: '-',
+            Result.DOUBLE_FORFEIT: '-',
+            Result.PENALTY_LL: '-',
+            Result.PENALTY_LD: '-',
+            Result.PENALTY_DL: '=',
+            Result.UNRATED_PENALTY_LL: '-',
+            Result.UNRATED_PENALTY_LD: '-',
+            Result.UNRATED_PENALTY_DL: '=',
```

Comment:
I'm not sure those are the right results to display.
`PENALTY_LD` is rated Loss for white and rated Draw for black, while `PENALTY_DL` is rated Draw for white and rated Loss for black.
Doesn't it depend on the colour, or did I miss something in the structure of the code?